src/test/util_tests.cpp

```diff
@@ -89,6 +112,14 @@ BOOST_AUTO_TEST_CASE(util_check)
     // Check that Assume can be used as unary expression
     const bool result{Assume(two == 2)};
     Assert(result);
+
+    // Check that Assert doesn't require copy/move
+    NoCopyOrMove x{9};
+    Assert(x).i += 3;
+    Assert(x).test();
```

Comment:
Maybe check the return value, if that was the point?

Reply:
Point was mostly to call `test()`